src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,694 @@
+
+If you want to use the SCIM API in a custom application or for ad-hoc requests, proceed to learn how to connect to the SCIM API.
+
+## Connect to the SCIM API [#connect]
+
+The SCIM API is available at `https://scim-provisioning.service.newrelic.com/scim/v2` and this URL is viewable in the Authentication Domain settings page. To access the SCIM API, your client must include a **bearer token** with each request. The token is displayed after saving your Authentication Domain configuration.
+
+If you are using a third-party identity provider, configure it to use **Bearer Token authorization** and plug in your API access token. Refer to your identity provider's documentation for help configuring this. Once configured, you're all set to import users and groups!
+
+The SCIM 2.0 protocol requires user and group data to be sent in accordance with a few core _schemas_ and encoded in JSON. See [RFC 7643 - SCIM Core Resources and Extensions](https://datatracker.ietf.org/doc/html/rfc7643#section-4) and [RFC 7643 - JSON Representation](https://datatracker.ietf.org/doc/html/rfc7643#section-8) for the specifics. Refer to [RFC 7644 - SCIM Protocol](https://datatracker.ietf.org/doc/html/rfc7644) for more information about the protocol used in this
+tutorial.
+
+For all requests to the SCIM API, you must provide the bearer token in an `Authorization` header. Here's an example with `curl`:
+
+```
+curl -X 'GET' -H 'Accept: application/json' -H "Authorization: Bearer $TOKEN" 'https://scim-provisioning.service.newrelic.com/scim/v2/Users'
+```
+
+Any request in the rest of this tutorial will receive a **401 Unauthorized** response if the API access token is missing or invalid.
+
+Example response:
+
+<Collapser title="401 Unauthorized">
+  ```
+  {
+    "schemas": [
+      "urn:ietf:params:scim:api:messages:2.0:Error"
+    ],
+    "detail": "invalid authentication token",
+    "status": "401"
+  }
+  ```
+</Collapser>
+
+## Create users in your Authentication Domain [#create-users]
+
+If you are using a third-party identity provider, refer to their documentation for how to push users with SCIM.
+
+Otherwise, send a `POST` request to `/scim/v2/Users` to create a user. The following user attributes are **required**:
+* `userName` This identifier **must** be unique within your Authentication Domain. Use the user's email address. 
+* `emails` Email address of the user.
+* `active` Boolean indicating whether or not the user should be active or inactive within New Relic.
+
+It is recomended to provide the following attributes for the best user experience:
+* `name.givenName` The user's first or given name.
+* `name.familyName` The user's last or family name.
+* `timezone` The user's timezone in IANA Time Zone database format.
+
+Take note of the returned user `id`. To update a user in the future you will need to supply the same ID with the request.
```

Comment:
This should be a <Callout> or something. Otherwise the curl seems to be related to it, but the curl is _actually_ more related to the bits above.

Or maybe just make a new section

```suggestion
Take note of the returned user `id`. To update a user in the future you will need to supply the same ID with the request.

### Example using `curl`
```